### PR TITLE
[2.7] PANDARIA: add tke-operator chart

### PR DESCRIPTION
**Pull request by GGGitBoy**
**Relate issue**
https://github.com/cnrancher/pandaria/issues/2633

----

请按照以下列表检查 PR 是否符合规范：

- [x] `Chart.yaml` 中是否配置 `catalog.cattle.io/rancher-version`
- [x] `Chart.yaml` 中是否配置 `catalog.cattle.io/kube-version`
- [x] 是否支持 `systemDefaultRegistry` 设置
- [x] 容器镜像是否为 Manifest
- [x] `values.yaml` 中容器镜像是否可以在 Rancher 构建镜像列表时被识别
    ```yaml
    repository: cnrancher/mirrored-image-name
    tag: v1.2.3
    ```

### CODE REVIEW

File: packages/rancher-tke-operator/100.0.0+up1.0.1/values.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+noProxy: ""
+additionalTrustedCAs: false
+lockName: ""
+lockNamespace: ""
+qps: ""
+burst: ""
+
+
+
+
```
Comment (Jason-ZW) [inline packages/rancher-tke-operator/100.0.0+up1.0.1/values.yaml:20]: Remove useless lines

**Review by STARRY-S — APPROVED** [review]
LGTM

File: charts/rancher-tke-operator/100.0.0+up1.0.1/values.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+noProxy: ""
+additionalTrustedCAs: false
+lockName: ""
+lockNamespace: ""
+qps: ""
+burst: ""
+
+
+
+
```
Comment (Jason-ZW) [inline charts/rancher-tke-operator/100.0.0+up1.0.1/values.yaml:20]: 同样问题

**Review by Jason-ZW — CHANGES_REQUESTED** [review]
Others LGTM

**Comment by STARRY-S** [comment]
Reverted since request changes were submitted after this PR was merged.

**Pull request merged**